front/kubernetes/staging/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+data:
+  auth: YTokYXByMSRWdVhZTHVFSiRVMHZhZHJjUUw3YmJsZjdFRG1DUUIx
```

Comment:
correct me if I'm wrong but we should probably not have secrets on Github?

Reply:
it's already the encrypted password (+username) as a base64 encoded string. so it's kind of ok to check it in.